src/Authentication/Authentication.Core/Utilities/AuthenticationHelpers.cs

```diff
@@ -113,23 +121,46 @@ private static async Task<InteractiveBrowserCredential> GetInteractiveBrowserCre
         {
             if (authContext is null)
                 throw new AuthenticationException(ErrorConstants.Message.MissingAuthContext);
-            var interactiveOptions = IsWamSupported() ? new InteractiveBrowserCredentialBrokerOptions(WindowHandleUtlities.GetConsoleOrTerminalWindow()) : new InteractiveBrowserCredentialOptions();
+            var interactiveOptions = IsWamSupported() ?
+                                        new InteractiveBrowserCredentialBrokerOptions(WindowHandleUtlities.GetConsoleOrTerminalWindow()) :
```

Comment:
If you are an SDK, you should expose the window handle. You can't assume the consuming app is a command line app.

Reply:
The windows handle behavior isn't changing with this PR. That said, we are intentionally assuming that this *IS* a command-line app. The "MS Graph PowerShell SDK" is only meant to be used from within a PowerShell language session, and is not intended to interact with graphical applications. There is an MS Graph SDK .NET SDK that can be used for graphical applications. For a GUI app to use this "SDK," it would also need to load the PowerShell runtime as part of the process, which would not really make sense and is not done. I do think the name "SDK" is confusing here from a product standpoint, but that is more reflective of its auto-generated nature rather than it being used as an SDK for .net. It really is just limited to PowerShell's runtime / command-line environment.